addons/mail/static/src/components/message/message_tests.js

```diff
@@ -1101,7 +1101,7 @@ QUnit.test('rendering of tracked field with change of value from 0 to non-0', as
     await this.createMessageComponent(message);
     assert.strictEqual(
         document.querySelector('.o_Message_trackingValue').textContent,
-        "Total:01",
+        "Total:1",
```

Comment:
Well this is not correct, we still want to display 0 of course.

Reply:
The only target of the PR is "false" that acts as "undefined" in JS (when a field is not set in python).